packages/office-ui-fabric-react/src/components/SpinButton/SpinButton.classNames.ts

```diff
@@ -19,7 +18,6 @@ export const getClassNames = memoizeFunction(
     styles: ISpinButtonStyles,
     disabled: boolean,
     isFocused: boolean,
-    keyboardSpinDirection: KeyboardSpinDirection,
```

Comment:
Use `@deprecated` instead and leave a comment that this was unused.

Reply:
This is internal code and safe to remove them.